src/daemon/ca/keys.rs

```diff
@@ -107,11 +107,11 @@ impl CertifiedKey {
 
         let not_after = self.incoming_cert().cert().validity().not_after();
 
-        let now = Time::now().timestamp_millis();
-        let until_not_after_millis = not_after.timestamp_millis() - now;
-        let until_new_not_after_millis = new_not_after.timestamp_millis() - now;
+        let now = Time::now().timestamp();
```

Comment:
Regarding the comment above, which wasn't changed as part of this work but with the goal of ensuring everything is correct and clear:

1. What does "we probably should know.." mean? Should know what? Did that mean to say "because we probably should _you_ know" as in we should act because we've been asked to?
2. Is it a) or b), or is it a) and b)?
3. Should the comment be updated to mention the 10% checks?
4. Should the comment, and the comment just below as well, be updated to refer to the new remaining/eligibility based naming?

Reply:
fixed